src/components/services/apiService.tsx

```diff
@@ -0,0 +1,10 @@
+export async function getData() {
+  try {
+    let request = await fetch('https://fakestoreapi.com/products');
```

Comment:
if axios is already installed , then replace fetch with axios

Reply:
replaced fetch with Axios.